src/com/mjcode/model/Piece.java

```diff
@@ -27,16 +35,56 @@ public Piece(Player player, Tile tile, int direction) {
      * @return true if the supplied piece belongs to a player's piece, else false
      */
     public boolean canCapture(Piece other) {
-        return false;
+        return other.getPlayer() != this.player;
     }
-
+    /*
     /**
      * The canMoveTo method determines if a pieces can legally move to a specified tile
      * @param tile The specified tile
      * @return True if piece can legally move to that tile, else false
      */
-    public boolean canMoveTo(Tile tile) {
-        return false;
+    /*public boolean canMoveTo(Tile tile) {
+        // Check if tile is occupied by the opponent's piece
+        if (tile.isOccupied()) {
+            return false;
+        }
+        // If not a King
+        if (!isKinged()) {
+            // If not a King
+            return true;
+        } else {
+            return false;
+
+        }
+    }*/
+    // Test method
+    public boolean canMoveTo(int newRow, int newCol) {
+        // Check if tile is occupied by the opponent's piece
+        if (tile.isOccupied()) {
+            return false;
+        }
+
+        // Calculate the direction of the piece based on the opponent's side
+        this.direction = (newRow > row) ? 1 : -1;
+
+        /*  If the piece is not Kinged, it can legally move one space diagonally
+            towards the opponent's side
+            */
+        if (!isKing) {
+            // Check if the move is one space diagonally in the specified direction
+            if (Math.abs(newRow - this.row) == 1 && newCol - this.col == 1) {
```

Comment:
- The calculation for Row should equal this Piece's direction, and it should not be the absolute value.
- The calculation for Column should be the absolute value.